setup.py

```diff
@@ -44,9 +44,9 @@ def run(self):
     license="BSD",
     platforms="Linux, Mac OS X",
     keywords=['helm', 'kubernetes'],
-    python_requires=">=3.4",
+    python_requires=">=3.6",
     install_requires=[
-        'ruamel.yaml>=0.15',
+        'ruamel.yaml<=0.15.54',
```

Comment:
dependencies shouldn't be pinned down

Reply:
this might be a relic from a few months back -- updates to the `ruamel` library tend to cause unpredictable behavior at times

Reply:
Any time that happens would be a bug to be fixed. New releases of ruamel.yaml shouldn't force new releases of chartpress. If pin-down is desired, it should go in a given project's CI requirements.txt, never a library's strictly enforced dependencies.